PowerUp/app/src/main/java/powerup/systers/com/CompletedScene.java

```diff
@@ -20,8 +20,16 @@ public void onClick(View v) {
 				Intent myIntent = new Intent(CompletedScene.this,
 						MapActivity.class);
 				startActivityForResult(myIntent, 0);
+				finish();
 			}
 		});
 	}
-	
+
+	@Override
+	public void onBackPressed() {
+		super.onBackPressed();
```

Comment:
You shouldn't be calling `super.onBackPressed()`. You don't want it to act the way it did before! :)